spec/services/wallet_transactions/create_service_spec.rb

```diff
@@ -76,7 +76,7 @@
 
     it 'enqueues a SendWebhookJob for each wallet transaction' do
       expect do
-        create_service.call
+        create_service
```

Comment:
`create_service` is actually the result here. `.call` worked because the result was an openstruct but now it's a proper BaseResult and breaks 😎

https://github.com/getlago/lago-api/blob/4712e76cce61f8aee375caebc5d32da643fb3273/spec/services/wallet_transactions/create_service_spec.rb#L6